promgen/templates/base.html

```diff
@@ -1,83 +1,88 @@
-  </script>
-  <script type="text/x-template" id="exporter-test-template">
-    {% include 'promgen/vue/exporter_test.html' %}
-  </script>
-  <script type="text/x-template" id="promql-query-template">
-    {% include 'promgen/vue/promql_query.html' %}
-  </script>
-  <script type="text/x-template" id="silence-form-template">
-    {% include 'promgen/vue/silence_form.html' %}
-  </script>
-  <script src="{% static 'js/promgen.js' %}"></script>
-  <script src="{% static 'js/mixins.vue.js' %}"></script>
-  <script src="{% static 'js/promgen.vue.js' %}"></script>
-  {% block javascript %}{% endblock %}
+    <script src="{% static 'js/jquery.min.js' %}"></script>
+    <script src="{% static 'js/bootstrap.min.js' %}"></script>
+    <script src="{% static 'js/bootstrap-switch.min.js' %}"></script>
+    <script src="{% static 'js/luxon.min.js' %}"></script>
+    <script src="{% static 'js/linkify.min.js' %}"></script>
+    <script src="{% static 'js/linkify-string.min.js' %}"></script>
+    
+    <script type="text/x-template" id="bootstrap-panel-template">
+      {% include 'promgen/vue/bootstrap_panel.html' %}
+    </script>
+    <script type="text/x-template" id="exporter-result-template">
+      {% include 'promgen/vue/exporter_result.html' %}
+    </script>
+    <script type="text/x-template" id="exporter-test-template">
+      {% include 'promgen/vue/exporter_test.html' %}
+    </script>
+    <script type="text/x-template" id="promql-query-template">
+      {% include 'promgen/vue/promql_query.html' %}
+    </script>
+    <script type="text/x-template" id="silence-form-template">
+      {% include 'promgen/vue/silence_form.html' %}
+    </script>
+
+    <script src="{% static 'js/promgen.js' %}"></script>
+    <script src="{% static 'js/mixins.vue.js' %}"></script>
+    <script src="{% static 'js/promgen.vue.js' %}"></script>
 
-  <datalist style="display:none" id="common.labels">
-    <option>severity</option>
-  </datalist>
+    {% block javascript %}{% endblock %}
 
-  <datalist style="display:none" id="common.annotations">
-    <option>summary</option>
-    <option>grafana</option>
-    <option>runbook</option>
-  </datalist>
+    <datalist style="display: none" id="common.labels">
+      <option>severity</option>
+    </datalist>
 
-  <datalist style="display:none" id="common.labels.severity">
-    <option>critical</option>
-    <option>major</option>
-    <option>minor</option>
-    <option>debug</option>
-  </datalist>
+    <datalist style="display: none" id="common.annotations">
+      <option>summary</option>
+      <option>grafana</option>
+      <option>runbook</option>
+    </datalist>
 
-  <script>app.mount("#vue")</script>
-</body>
+    <datalist style="display: none" id="common.labels.severity">
+      <option>critical</option>
+      <option>major</option>
+      <option>minor</option>
+      <option>debug</option>
+    </datalist>
 
+    <script>
+      app.mount("#vue");
+    </script>
+  </body>
```

Comment:
> Updated this file's format, so this file has a lot of changes.
> The main changes of base.html is to move the import of vuejs to the top of this file.

If you want to place the Vue imports at the top of this file, just do that.
If you want to reformat the file, then do it in a separate commit.

Both things are unrelated and don't depend on each other, so they should not be in the same commit. That makes the `git diff` unnecessarily bigger, and harder to review when looking for bugs.

Reply:
It looks like you have addressed this comment in the following commits:
- 43381cccbaa8ee644d3c72c043c22c552fec6586
- ee0b9c925ac14b7a152c00e9b6c48f3a7bb1e0e7
- a3677058a4338f0497c9bccdfd7176bbdc13b3c5

This conversation is resolved.